release/README.md

```diff
@@ -56,6 +56,10 @@
 * Copy the release notes you have written before and paste them in the Earthly Community slack channel `#announcements`, together with a link to the release's GitHub page. If you have Slack markdown editing activated, you can copy the markdown version of the text.
 * Ask Adam to tweet about the release.
 
+### One-Time (clear this section when done during release)
+
+* Update the website with the documentation for bootstrap, include `--with-autocomplete` as part of the installation.
```

Comment:
I think it's fine to not include this flag for the mac installation since the autocomplete is done through homebrew. We could remove sudo for mac then.

Reply:
You helped me find a bug that would have broken autocomplete on macOS again. That is included here and an addendum added to this point.